app/src/main/java/org/fossasia/pslab/fragment/ControlFragmentAdvanced.java

```diff
@@ -83,6 +85,85 @@ public void onClick(View v) {
         etwidgetControlAdvanced10.init(getContext(), 0.1, 0.0, 1.0);
         etwidgetControlAdvanced11.init(getContext(), 1.0, 0.0, 360.0);
 
+        buttonControlAdvanced1.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
```

Comment:
```NumberFormatException``` what if string given to Double.valueOf isn't decodeable as a double. [this exception](https://github.com/fossasia/pslab-android/blob/master/app/src/main/java/org/fossasia/pslab/adapters/ControlMainAdapter.java#L114), Android Studio doesn't tell about all possible errors.